core/src/main/scala/stryker4s/mutants/findmutants/FileCollector.scala

```diff
@@ -1,29 +1,87 @@
 package stryker4s.mutants.findmutants
 
 import better.files._
+import grizzled.slf4j.Logging
 import stryker4s.config.Config
+import stryker4s.run.process.{Command, ProcessRunner}
+
+import scala.util.{Failure, Success}
 
 trait SourceCollector {
   def collectFilesToMutate(): Iterable[File]
+  def filesToCopy(processRunner: ProcessRunner): Iterable[File]
 }
 
-class FileCollector(implicit config: Config) extends SourceCollector {
+class FileCollector(implicit config: Config) extends SourceCollector with Logging {
+
+  /**
+    * Get path separator because windows and unix systems have different separators.
+    */
+  private[this] val pathSeparator = config.baseDir.fileSystem.getSeparator
 
+  /**
+    *  Collect all files that are going to be mutated.
+    */
   override def collectFilesToMutate(): Iterable[File] = {
-    filesToMutate.filterNot(file => filesToExcludeFromMutation.contains(file))
+    filesToMutate
+      .filterNot(filesToExcludeFromMutation.contains(_))
+      .filterNot(stryker4sTmpFiles.contains(_))
   }
 
-  private[this] val filesToMutate: Seq[File] = {
-    config.mutate
-      .filterNot(file => file.startsWith("!"))
+  /**
+    * Collect all files that are needed to be copied over to the Stryker4s-tmp folder.
+    *
+    * Option 1: Copy every file that is listed by git.
+    * Option 2: Copy every file that is listed by the 'files' config setting
+    * Option 3: Copy every file in the 'baseDir' excluding target folders.
+    */
```

Comment:
I think the order here should be different. If the user has specified `files`, use that. If not, use git, if not, option 3. This way the user can still override any problems they come into with the git listing